### ISSUE TITLE
Incorrect DragOverlay position when 'overflow' is set on html tag

### ISSUE THREAD

**Issue by alexeypt** [clauderic/dnd-kit#378]
Hello, I bumped into a really weird issue.

We use on our project the following global styles in order to prevent shifting issue when vertical scrollbar appears:
```
html {
    // Fixing the layout shift problem when the scrollbar is visible (https://aykevl.nl/2014/09/fix-jumping-scrollbar)
    width: 100vw;
    overflow-x: hidden;
}
```

Surprisingly but it affects `dnd-kit`. I created simple code sandbox (but I also reproduced it inside `dnd-kit` storybook): https://codesandbox.io/s/dnd-kit-sortable-forked-f35fi (issue is reproducible when you drag-and-drop items at the bottom).

I debug `dnd-kit` code a bit and found initial issues's cause. Below is the code for getting all scrollable ancestors:
```
function isScrollable(node) {
  const computedStyle = window.getComputedStyle(node);
  const overflowRegex = /(auto|scroll|overlay)/;
  const properties = ['overflow', 'overflowX', 'overflowY'];
  return properties.find(property => {
    const value = computedStyle[property];
    return typeof value === 'string' ? overflowRegex.test(value) : false;
  }) != null;
}

function getScrollableAncestors(element) {
  const scrollParents = [];

  function findScrollableAncestors(node) {
    if (!node) {
      return scrollParents;
    }

    if (node instanceof Document && node.scrollingElement != null) {
      scrollParents.push(node.scrollingElement);
      return scrollParents;
    }

    if (!(node instanceof HTMLElement) || node instanceof SVGElement) {
      return scrollParents;
    }

    if (isScrollable(node)) {
      scrollParents.push(node);
    }

    return findScrollableAncestors(node.parentNode);
  }

  return element ? findScrollableAncestors(element.parentNode) : scrollParents;
}
```

Adding `overflow-x: hidden` to `html` tag leads to the fact that `window.getComputedStyle(document.querySelector('html'))` returns `overflow-y: auto` (even if you manually set `overflow-y: visible` it returns  `overflow-y: auto` in computed styles). As result, 
`overflowRegex.test(value)` returns `true` and `offsetTop` is counted. The next `element.parentNode` is `document` (last node, end of recursion) and the same `offsetTop` is counted again.

**Comment by hemengke1997** [comment]
https://github.com/clauderic/dnd-kit/pull/379#issue-695499950

hey, you can set measure function by `measuring.droppable.measure`

**Comment by clauderic** [comment]
I believe this is fixed by #430, feel free to re-open if not.

https://codesandbox.io/s/dnd-kit-sortable-forked-jlscc
